src/types/types.js

```diff
@@ -881,6 +884,7 @@ export type EdgeSwapQuote = {
   +fromNativeAmount: string,
   +toNativeAmount: string,
   +networkFee: EdgeNetworkFee,
+  +swapPluginType?: EdgeSwapPluginType,
```

Comment:
Would it make more sense to just put the `EdgeSwapInfo` here, to provide complete information about the chosen plugin? Otherwise, the `pluginId` should be enough to go look this up, without copying the fields over.

Reply:
I originally wanted that but I considered that the pluginType might actually be something that slightly mismatches the quote. ie 'hybrid' plugin type that can do 'DEX' or 'CEX' or a 'hybrid' swap if we were to route through Changenow then Uniswap. Admittedly this begs the question of whether the quote should use a name `EdgeSwapPluginType` vs `EdgeSwapType`. The EdgeSwapInfo is for helping the GUI display plugins while the quote is useful for the matching engine.

This is all super nuanced and maybe overthinking so if you think this flexibility is over-architecting, then I'm 100% fine just plopping the EdgeSwapInfo into the quote.

Reply:
Well, the main thing is that we already have `pluginId`, so if we add more stuff like hybrid routing, we would have to deprecate / edit / adjust the `pluginId` as well. So, we can rely on the `pluginId` / `pluginInfo` to give us the information for now, and then cross the hybrid bridge when we come to it. For instance, maybe we define `pluginId: "hybrid"` to mean "go look elsewhere for this information".

Reply:
Ok. i'll just add the swapInfo